boards/shields/nrf2240ek/nrf2240ek.conf

```diff
@@ -0,0 +1,2 @@
+CONFIG_BOOT_BANNER=n
```

Comment:
Same question as for nRF2220: are these options necessary for nRF52 series too?

Reply:
This configuration is needed unconditionally for nRF53 and in some cases for nRF52 too. After F2F discussion we decided to keep it as a limitation.